keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/PlaceHolderDoubleVertex.java

```diff
@@ -0,0 +1,19 @@
+package io.improbable.keanu.vertices.dbl.nonprobabilistic;
+
+import io.improbable.keanu.tensor.dbl.DoubleTensor;
+import io.improbable.keanu.vertices.NonProbabilistic;
+import io.improbable.keanu.vertices.dbl.DoubleVertex;
+import io.improbable.keanu.vertices.dbl.KeanuRandom;
+
+public class PlaceHolderDoubleVertex extends DoubleVertex implements NonProbabilistic<DoubleTensor> {
```

Comment:
Can't you just use a `ConstantDoubleVertex` and set its value?

Reply:
Could but that's semantically wrong I think. It would save use creating new classes, but it'd be confusing.